### PR TITLE
docs(readme): `toggler` doesn't defines mappings in visual mode

### PR DESCRIPTION
**Pull request by refaelsh**
This PR will solve this issue: https://github.com/numToStr/Comment.nvim/issues/195.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Following are the **default** config for the [`setup()`](#setup). If you want to
         block = 'gbc',
     },
 
-    ---LHS of operator-pending mappings in NORMAL + VISUAL mode
+    ---LHS of operator-pending mappings in NORMAL mode
```
Comment (numToStr) [inline README.md:94]: This is not right. `opleader` also defines the visual mappings.

File: README.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ Following are the **default** config for the [`setup()`](#setup). If you want to
         eol = 'gcA',
     },
 
-    ---Create basic (operator-pending) and extended mappings for NORMAL + VISUAL mode
+    ---Create basic (operator-pending) and extended mappings for NORMAL mode
```
Comment (numToStr) [inline README.md:114]: same here.

**Review by numToStr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (refaelsh) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (refaelsh) [reply README.md]: Done.

**Review by numToStr — APPROVED** [review]

**Pull request merged**